### ISSUE TITLE
Customizeable actions?

### ISSUE THREAD

**Issue by Anachron** [dylanaraps/fff#15]
Can we make (for example) the trash operation customizable?

I use a custom written trash command.

For now it is hardcoded:
https://github.com/dylanaraps/fff/blob/master/fff#L71

Also wondered if I could easy add my own actions? I would love to add "send-by-mail" and alike without modifying this source too much.

If you say this is going to kill the purpose of `fff` I am fine with that as well.

Thanks and have a nice day!

**Comment by dylanaraps** [comment]
This won’t be hard to do at all (I’ve got it almost working locally in a separate branch).

I’ll get to it.

**Comment by RobSis** [comment]
I'd also love to see this feature. Would be great if you could also unbind the default actions.

Can you maybe push the branch on github?

I freaking love this tool btw, and I think it has a great potential to be used in scripts as abstract file selector similar to e.g. `dialog --fselect`.

**Comment by lwilletts** [comment]
Yes this would be fantastic, sometimes I want to open files directly in the browser but there's no way to specify custom commands to custom bindings.

**Comment by dylanaraps** [comment]
I've been thinking about it for a while and haven't figured out the right way to do it. I will get to it soon.
